other/bootstrap_daemon/README.md

```diff
@@ -50,23 +48,59 @@ Get your public key and check that the daemon initialized correctly:
 sudo grep "tox-bootstrapd" /var/log/syslog
 ```
 
+##For `systemd` users:
+
+Place tox_bootstrap.service into /etc/systemd/system:
+```sh
+sudo cp tox_bootstrap.service into /etc/systemd/system/
+```
+
+Make sure, that path to `chown` and `chmod` is correct in `tox_bootstrap.service` (they may be different in some distributions, by default  `/bin/chown` and `/bin/chmod`) 
```

Comment:
1. The service file has different name
2. It doesn't use `chmod`
3. The defaults in the service file are not `/bin` (but should be).